clients/include/testing_axpy.hpp

```diff
@@ -17,6 +17,15 @@
 
 using namespace std;
 
+#define CLEANUP()                                                  \
```

Comment:
We could change this throughout the test code. It would require changing many other files, and i would likely be best to do it in a separate pull request.